Sofa/Component/MechanicalLoad/src/sofa/component/mechanicalload/DiagonalVelocityDampingForceField.h

```diff
@@ -44,12 +44,22 @@ class DiagonalVelocityDampingForceField : public core::behavior::ForceField<Data
     typedef core::objectmodel::Data<VecCoord> DataVecCoord;
     typedef core::objectmodel::Data<VecDeriv> DataVecDeriv;
 
-    /// air drag coefficient.
-    Data< VecDeriv > dampingCoefficients;
+    union
+    {
+        /// velocity damping coefficients (by cinematic dof)
+        Data< VecDeriv > d_dampingCoefficients;
+
+        SOFA_ATTRIBUTE_DEPRECATED("v23.12", "v24.06", "This Data is now replaced by d_dampingCoefficients")
```

Comment:
The dampintCoeficients data is deprecated in favor of d_dampingCoefficients, during the deprecation phase (before final removal) it is possible to do in constructor dampingCoefficients.setParent(d_dampingCoefficients) so the old data field is still "functionning".

Reply:
Here it is not necessary because of the union

Reply:
I will need some time to think if using union for that purpose is a wise move or one an act of c++hijacking